@AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/core/fn_coreUpdate.sqf

```diff
@@ -21,7 +21,10 @@ params ["_heli"];
 
 if (isGamePaused) exitwith {};
 
-private _deltaTime = ((["sfmplus_deltaTime"] call BIS_fnc_deltaTime) min 1/30);
```

Comment:
This File change is duplicated in the Tail rotor Pr

Reply:
Removing them would break this PR